docs/usage.rst

```diff
@@ -0,0 +1,719 @@
+Usage
+=====
+
+The aim of
+:mod:`audbackend`
+is to provide an
+abstract interface for
+any kind of file storage system.
+Even those,
+that have not been
+invented yet :)
+
+This tutorial is divided
+into two parts.
+In the first part,
+we show the basic usage
+by means of a
+standard file system.
+In the second part,
```

Comment:
As for basics, I would directly link to the section by adding

```rst
.. _development:
```
in front of the sub-section and changing here to

```suggestion
Under :ref:`development`,
```